arbor/fvm_layout.hpp

```diff
@@ -112,7 +113,7 @@ fvm_cv_discretization& append(fvm_cv_discretization&, const fvm_cv_discretizatio
 // Construct fvm_cv_discretization from one or more cells.
 
 fvm_cv_discretization fvm_cv_discretize(const cable_cell& cell, const cable_cell_parameter_set& global_dflt);
-fvm_cv_discretization fvm_cv_discretize(const std::vector<cable_cell>& cells, const cable_cell_parameter_set& global_defaults);
+fvm_cv_discretization fvm_cv_discretize(const std::vector<cable_cell>& cells, const cable_cell_parameter_set& global_defaults, const arb::execution_context& ctx);
 
```

Comment:
Here and below: if we give the context parameter a default value (e.g. a default-constructed `execution_context`) I believe we could avoid the modifications to the unit tests (save for any parallel tests of course!)

Reply:
Okay, I will make the change :)